README.md

```diff
@@ -17,75 +17,81 @@ This is a general purpose library for working with Content Security Policy polic
 mvn install
 ```
 
+### A Note on CSP
+
+The CSP specification is fairly complex even if you only care about the latest version. However, in practice you are likely to care that your policy does the things you intend it to on the browsers you care about, which are likely to implement different and potentially broken subsets of the specification (and in some cases supersets). And there are inevitable tradeoffs to be made regarding the size of your policy vs the security it provides.
```

Comment:
> (and in some cases supersets)

Not supersets. I don't know what the word for this relationship would be though.

Reply:
Will change to "and potentially behavior which is not in the specification".